### DISCUSSION TITLE
Pull Request Dashboard shows count status of approved vs. not

### DISCUSSION THREAD

**Discussion by peterkline** [community/community#72966] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

In some repos we require multiple reviewers and others only 1.  While we only require one reviewer we really want everyone added to review/approve before we merge.  

If would be nice if the `pulls` page showed a PR as approved but also a counter
Something like 1/1 approved or 1/4 approved so that we know that on the 1 of 4 reviewed we need to get the other three to take a look.

**Comment by dletter** [comment]
Making my upvote bigger here ;). Really would like this on the pulls page, next to number of tasks done.
